### ISSUE TITLE
Adding additional users to the RMM

### ISSUE THREAD

**Issue by rtwright68** [wh1te909/tacticalrmm#52]
I'm sure I'm missing the obvious.  How do we go about adding a new user for the RMM?
Is that a CLI process?

**Comment by rtwright68** [comment]
Got some help to get a user added so I have the steps.  Hoping that will be added to the GUI at some point?  :-)

**Comment by wh1te909** [comment]
great! I'll add the instructions here for visibility in case someone else has question. And yes @sadnub is going to start working on adding user management to the gui.

```
cd /rmm/api/tacticalrmm
source ../env/bin/activate
python manage.py createsuperuser --username <USERNAME> --email <EMAIL>
python manage.py generate_totp # copy and paste this code into the next line
python manage.py generate_barcode <CODE> <USERNAME> <yourwebsite.com>
deactivate
```

**Comment by wh1te909** [comment]
this has been implemented in 098960009df1c673e0a9586421f9d61e5ce2d71a
Settings > User Administration

![users](https://user-images.githubusercontent.com/7434746/90852102-78529b80-e32b-11ea-9146-1633abf1b934.png)
